application/controllers/cron/Project_status_stats.php

```diff
@@ -22,6 +22,11 @@ public function __construct()
 
 	public function author()
 	{
+		$this->db->query('
```

Comment:
I think a comment here would be necessary - I had to look at the commit message to understand _why_ this is being done.